tests/foreman/ui/test_gpgkey.py

```diff
@@ -384,7 +380,118 @@ def test_consume_content_1(self):
 
         """
 
-        pass
+        product_name = gen_string("alpha", 8)
+        repository_name = gen_string("alpha", 8)
+        activation_key_name = gen_string("alpha", 8)
+        key_content = read_data_file(ZOO_CUSTOM_GPG_KEY)
+        # step1: Create gpg-key
+        gpgkey_attrs = entities.GPGKey(
+            name=key_name,
+            content=key_content,
+            organization=self.org_id
+        ).create_json()
+        # step 1.2: Create new lifecycle environments
+        lifecycle_env = entities.LifecycleEnvironment(
+            organization=self.org_id
+        ).create_json()
```

Comment:
The only attribute used from `lifecycle_env` is its ID. This can be changed to something like the following:

```python
lc_env_id = entities.LifecycleEnvironment(
    organization=self.org_id
).create_json()['id']
```

Then, adjust the two usages of `lifecycle_env['id']`.